src/nvim/CMakeLists.txt

```diff
@@ -526,11 +526,6 @@ endfunction()
 
 set(NO_SINGLE_CHECK_HEADERS
   os/win_defs.h
```

Comment:
I don't think this one can be checked at all, so @ZyX-I I think we can close https://github.com/neovim/neovim/issues/5321 ?